tests/pipelines/text_to_video_synthesis/test_text_to_video_zero_sdxl.py

```diff
@@ -0,0 +1,235 @@
+            expected_slice2 = np.array([0.66, 0.49, 0.40, 0.69, 0.47, 0.51, 0.73, 0.65, 0.52])
+
+        assert np.abs(first_frame_slice.flatten() - expected_slice1).max() < 1e-2
+        assert np.abs(last_frame_slice.flatten() - expected_slice2).max() < 1e-2
+
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_attention_slicing_forward_pass(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_dict_tuple_outputs_equivalent(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_float16_inference(self):
+        pass
+
+    @unittest.skip(reason="Batching needs to be properly figured out first for this pipeline.")
+    def test_inference_batch_consistent(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_inference_batch_single_identical(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_model_cpu_offload_forward_pass(self):
+        pass
+
+    @unittest.skip(reason="`num_images_per_prompt` argument is not supported for this pipeline.")
+    def test_pipeline_call_signature(self):
+        pass
+
+    @unittest.skip(reason="`num_images_per_prompt` argument is not supported for this pipeline.")
+    def test_save_load_float16(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_save_load_local(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_save_load_optional_components(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_sequential_cpu_offload_forward_pass(self):
+        pass
+
+    @unittest.skip(reason="Cannot call `set_default_attn_processor` as this pipeline uses a specific attention processor.")
+    def test_xformers_attention_forwardGenerator_pass(self):
+        pass
+
+
+@nightly
+@require_torch_gpu
+class TextToVideoZeroSDXLPipelineSlowTests(unittest.TestCase):
+    def test_full_model(self):
+        model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+        pipe = self.pipeline_class.from_pretrained(
+            model_id, torch_dtype=torch.float16, variant="fp16", use_safetensors=True
+        ).to("cuda")
+        pipe.scheduler = DDIMScheduler.from_config(pipe.scheduler.config)
+        generator = torch.Generator(device="cuda").manual_seed(0)
```

Comment:
```suggestion
        generator = torch.Generator(device="cpu").manual_seed(0)
```

Reply:
this is a slow test and is supposed to run on GPU. why should we change the generator device to cpu?

Reply:
Using a generator set to CPU helps make the noise generation process more deterministic. The resulting noise tensor is then moved to GPU.